client/packages/system/src/Name/Components/NameOptionRenderer/NameOptionRenderer.tsx

```diff
@@ -26,12 +26,12 @@ export const getNameOptionRenderer =
                 whiteSpace: 'no-wrap',
               }}
             >
-              {item.code}
+              {name.code}
             </Typography>
           </Box>
           <Typography>
-            {item.name}
-            {item.isOnHold ? ` (${onHoldLabel})` : ''}
+            {name.name}
```

Comment:
part of the reason for using `item` as the variable name is to avoid having `name.name` 🤷

Reply:
Hmm fair. I did it because I found it weird reading for the first while because the `item` table exists and has many of the same fields, though it was similar to the [`Flatlist` props I'm used to](https://reactnative.dev/docs/flatlist#required-renderitem). I had considered staying true to MUI prop names by calling it `option`, but thought continuing the `name` table specific context was useful. `optionName` was unaesthetic, 4 char word pleases me haha.